lib/src/dart.dart

```diff
@@ -151,6 +151,7 @@ Future<void> precompile({
   required String name,
   required String outputPath,
   required String packageConfigPath,
+  List<String> additionalSources = const [],
```

Comment:
Can we update the documentation comment?

Reply:
done (and in other files)